controllers/delete_pods_for_buggification_test.go

```diff
@@ -167,6 +167,7 @@ var _ = Describe("delete_pods_for_buggification", func() {
 				err = k8sClient.Delete(context.TODO(), pod)
 				Expect(err).NotTo(HaveOccurred())
 
+				pod.ResourceVersion = ""
```

Comment:
So technically the "right" way would actually be to do a get before modifying the object. I'll change the code since that's the better approach. Setting the `ResourceVersion` to `""` basically means use my changes and don't try to compare the versions.

Reply:
I correct myself the issue is that we get the resource and then we delete the resource and after that we want to create the resource with an already specified `ResourceVersion`, so we have to set the `ResourceVersion` to an empty string.